lib/spritesheet-templates.js

```diff
@@ -44,7 +44,16 @@ function templater(params, options) {
   var items = JSON.parse(JSON.stringify(params.items));
   var spritesheet = JSON.parse(JSON.stringify(params.spritesheet));
   spritesheet.name = options.spritesheetName || 'spritesheet';
+  
+  // Create a px namespace
+  var spritesheetPx = {};
+  spritesheet.px = spritesheetPx;
 
+  // For each of the height, width, add a px after that
+  ['height', 'width'].forEach(function (key) {
+    spritesheet.px[key] = spritesheet[key] + 'px';
+  });
```

Comment:
I think I might be missing something but isn't this covered by `templater.ensureTemplateVariables(spritesheet)` already?